### PR TITLE
Feature Branch for Microverse portfolio project 

### PR DESCRIPTION
**Pull request by Spacelama587**
This is the pull request for Portfolio: setup and mobile first
The project includes hero section for mobile view of the portfolio project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+                Hey There. <br />           
+                I’m Vijith.
+            </h1>
+            
+            <div class="supporting-text">
+            <h4 >
+                I'm a Software Developer
+            </h4>
+        
+        <p class="description">
```
Comment (ademibowale) [inline index.html:34]: - [ ] Its very essential and maandatory to stick to figma design ,i would suggest you to add  the backgrund image just like the orignal figma design to make your website look professional.
- [ ] your design
Screenshot 2023-05-31 190629

- [ ] figma template
figma temp

File: style.css [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+  padding: 0;
+}
+
+.navbar {
+  background-color: #3c3a39;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 70px;
+  width: 100%;
```
Comment (ademibowale) [inline style.css:12]: - [ ] I would kindly suggest you to align the the The hamburger button  and logo button you can add magin top to hambuger button to be at same top with the logp button .
- [ ] your design
Screenshot 2023-05-31 190629

- [ ] figma template
figma temp

File: style.css [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+
+.hero {
+  background: #1c1a19;
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  background-image: url(./images/illustration-header-mobile.svg);
+  background-repeat: no-repeat;
+  background-position: right bottom;
+}
```
Comment (ademibowale) [inline style.css:46]: - [ ] Its very essential and maandatory to stick to figma design ,i would suggest you to add  the backgrund image just like the orignal figma design to make your website look professional.
- [ ] your design
Screenshot 2023-05-31 190629

- [ ] figma template
figma temp

**Review by ademibowale — CHANGES_REQUESTED** [review]
## STATUS: Changes Required :recycle:

### Hi @DevaWinner , 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight 🥇

✅ Descriptive Pull Request title and summary. ✔️
✅ Good use of Github flow ✔️
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @Adebowale ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by NahnahAJ** [comment]
## Project Status: INVALIDATED :red_circle: 
Hi @Spacelama587 , 

While you made a great effort in this project, I can't go ahead and review your code.


## Invalid Code Review Request

- This is because you have submitted a project, that has some linter checks failing. So please fix them and ensure that all your linters are passing :+1: 
- Your linter checks
image


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @NahnahAJ in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+</head>
+<body>
+  <main>
+    <header class="navbar">
+        <div class="logo">
+            <a href="#"><img src="images/Logo Placeholder.svg"></a>
+        </div>
+        <div class="hamburger-menu">
+            <a href="#"><img src="images/Icon - Menu.svg" alt="hamburger-menu"></a>
+        </div>
```
Comment (Cabraham1) [inline index.html:21]: - [ ] I Kindly suggest you  Change the `<div>` tag to a `<nav>` tag that aligns with the project requirement and improves semantic markup for navigation. Good suggestion!

File: index.html [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+              <a href="#">
+                <img src="./images/twitter.svg" alt="twitter" />
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="./images/medium.svg" alt="medium" />
+              </a>
+            </li>
+          </ul>
```
Comment (Cabraham1) [inline index.html:67]: - [ ] To ensure the functionality of your social icons and enable visitors to access your designated social domains, I highly recommend updating the links.
- [ ] replaced the href attribute values (#) with the appropriate URLs for each social platform. You need to replace your_twitter_username, your_instagram_username, and your_linkedin_username with your actual usernames or profile URLs on those platforms.

**Review by Cabraham1 — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

## Hey @Spacelama587   :wave:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following GitHubflow
✔️ Following some requirements

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:
- [ ] I kindly request that you adhere to the Figma design provided for your project. The Figma design serves as a visual reference and guideline for implementing the desired user interface and overall look of your application. Please ensure that the implemented code aligns with the design specifications provided in Figma. This includes aspects such as layout, color scheme, typography, and any other visual elements defined in the design.
- [ ] Your Design
image

- [ ] Microverse Figma Design

image

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers, and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+</head>
+<body>
+  <main>
+    <header class="navbar">
+        <div class="logo">
+            <a href="#"><img src="images/Logo Placeholder.svg"></a>
+        </div>
+        <div class="hamburger-menu">
+            <a href="#"><img src="images/Icon - Menu.svg" alt="hamburger-menu"></a>
+        </div>
```
Comment (roshan-bajgain) [inline index.html:21]: - [ ] [Optional] I would suggest you please use the `nav` inplace of the `div` tag for semantics reasons and the right position for convention and user experience for mobile devices. As this requirement was not introduced by the first reviewer.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="./images/medium.svg" alt="medium" />
+              </a>
+            </li>
+          </ul>
+          </div>
+    </section>
```
Comment (roshan-bajgain) [inline index.html:70]: - [ ] [Optional] This is you last review so I am giving it an optional please implement it. I would suggest please describe the gap between the `p` tag and icons.
### Your design
his

**Review by roshan-bajgain — APPROVED** [review]
Hi @Spacelama587, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**